queries.sql

```diff
@@ -44,3 +44,13 @@ SELECT AVG(weight_kg) AS average_weight from ANIMALS;
 SELECT MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT MAX(weight_kg), MIN(weight_kg), species FROM animals GROUP BY species;
 SELECT AVG(escape_attempts), species FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* Day 3 */
+SELECT owners.full_name, owners.id, animals.name FROM animals INNER JOIN owners ON animals.owner_id = owners.id;
+SELECT animals.id, animals.name, species.name FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT owners.full_name, animals.name FROM animals RIGHT JOIN owners ON animals.owner_id = owners.id;
+SELECT COUNT(*), species.name FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
+SELECT animals.name FROM animals RIGHT JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name LIKE 'Jennifer Orwell';
```

Comment:
- [ ] This query should fetch all Digimon owned by jennifer Orwell but in your query code, i do not see any clause pointiong to `digimon`. See hint below 👇🏾 

```sql
SELECT animals.* FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   LEFT JOIN species
   ON animals.species_id = species.id
   WHERE owners.full_name = 'Jennifer Orwell'
   AND species.name = 'Digimon'; # see specie clause here
```